readme.md

```diff
@@ -170,6 +170,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [The PR/issues search box is wider.](https://user-images.githubusercontent.com/1402241/55069759-bceaf080-50bf-11e9-84d0-7707de2eb9e9.png)
 - [A warning appears when trying to create a PR from the default branch.](https://user-images.githubusercontent.com/1402241/52543516-3ca94e00-2de5-11e9-9f80-ff8f9fe8bdc4.png)
 - [A warning appears when unchecking `Allow edits from maintainers`.](https://user-images.githubusercontent.com/1402241/53151888-24101380-35ef-11e9-8d30-d6315ad97325.gif)
+- [Comment boxes no longer show scroll bars (they always fit their content)`.](https://user-images.githubusercontent.com/1402241/54336211-66fd5e00-4666-11e9-9c5e-111fccab004d.gif)
```

Comment:
This is not entirely true, in for example, the "Review changes" comment box, I get a scrollbar after a while, but I guess we can ignore that case for a simplified description. Just wanted to mention it.

Reply:
If that happens, I missed it. I'll look into this and the possible freeze

Reply:
We can't touch that at all. The textarea was overflowing the viewport and it's in the `position:fixed` header, so we can't scroll along it. I excluded it to avoid any issues with it (I caused a few bugs on that very box many times already)